waylay-sdk-registry-types/src/waylay/services/registry/models/deprecate_previous_policy.py

```diff
@@ -11,16 +11,23 @@
 
 from __future__ import annotations
 
-from enum import Enum
-
-
-class DeprecatePreviousPolicy(str, Enum):
-    """DeprecatePreviousPolicy."""
-
-    NONE = "none"
-    ALL = "all"
-    PATCH = "patch"
-    MINOR = "minor"
-
-    def __str__(self) -> str:
-        return str(self.value)
+from typing import (
+    Union,
+)
+
+from typing_extensions import (
+    Annotated,  # >=3.9
+)
+
+from ..models.deprecate_previous_policy_any_of import DeprecatePreviousPolicyAnyOf
+from ..models.deprecate_previous_policy_any_of1 import DeprecatePreviousPolicyAnyOf1
+from ..models.deprecate_previous_policy_any_of2 import DeprecatePreviousPolicyAnyOf2
+from ..models.deprecate_previous_policy_any_of3 import DeprecatePreviousPolicyAnyOf3
+
+DeprecatePreviousPolicy = Union[
```

Comment:
@plankthom I wonder what happened here, it used to be a nice enum?

(no need to hold back a release for this, but just a bit strange)

Reply:
it's a consequence of `anyOf` that was generated from the union type: 
https://github.com/waylayio/service-catalog/blob/d82adb3ecb769c46925dd87bb405774d9caf1c30/specs/cache/api.waylay.io/registry/v2/openapi.json#L414

Don't know why it has changed... The `anyOf` with `const` or `enum` allow for better documentation in the api docs though: https://docs-staging.waylay.io/openapi/sdk/redocly/registry.html#tag/Models/operation/create_models

Reply:
We could in our transformer check for `anyOf` that can be flattened to an `enum` ...